src/r/model.R

```diff
@@ -48,13 +49,35 @@ calculate_capacity_by_geography <- function(geography){
   return(aggregated_capacity)
 }
 
+calculate_demand_by_geography <- function(geography, GM_Primary_school_population_by_area_and_age){
+  
+  #could do this using aggregate function as below, but have used dplyr instead
+  #aggregated_population <- aggregate(GM_Primary_school_population_by_area_and_age$Population, by=list(Geography=GM_Primary_school_population_by_area_and_age$LA), FUN=sum)
+  #aggregated_population <- setnames(aggregated_population,"x","Demand")
+  if (geography == "ward") {
+    aggregated_population <- GM_Primary_school_population_by_area_and_age %>% 
+      group_by(Ward.Code, Ward.Name, Local.Authority) %>%
```

Comment:
The calculation on demand runs on LA or ward name while calculating for capacity runs on the ward or LA code. This needs to be rectified before we are able to combine the data in the next step. Need to decide whether codes or names work better